README.md

```diff
@@ -34,7 +33,7 @@ Where `opts` is an optional object with properties
  * `optimumvideowidth` [`Number`] The optimum width of the video itself, used when there are multiple video renditions available to
  decide which to display (the smallest one that's at least as large as this width, if it exists)
  * `placeholder` [`Boolean`] Show just the poster image, load (and play) video on click
- * `placeholdertitle` [`Boolean`] Show just the title as an overlay on the placeholder
+ * `placeholderInfo` [`Array`] A list of extra information to display on the placeholder (Available: title, description, brand, duration)
```

Comment:
I don't see any tests for the new features, `description`, `brand`, `duration`. Would you be able to add some?